modules/functional-test/src/main/resources-filtered/server-start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+APPLICATION_INSTRUMENT="-javaagent:/vagrant/target/org.jacoco.agent-${jacoco.version}-runtime.jar=destfile=/vagrant/target/jacoco/jacoco-it.exec,append=false"
+CONSOLE_LOG=wasabi-os-console.log
+MAIN_JAR=/vagrant/target/wasabi-main-*-all.jar
+
+JAVA_OPTIONS="-server -Xmx4096m \
+  ${APPLICATION_INSTRUMENT} \
+  -Dlogback.configurationFile=./logback.xml"
```

Comment:
"logging.properties" needs to be added. As one of the library is using java.util.logging